lib/src/barback/dartdevc/workers.dart

```diff
@@ -13,4 +13,8 @@ final analyzerDriver = new BazelWorkerDriver(() => Process.start(
     p.join(sdkDir.path, 'bin', 'dartanalyzer'),
     ['--build-mode', '--persistent_worker']));
 
+/// Manages a shared set of persistent dartdevc workers.
+final dartdevcDriver = new BazelWorkerDriver(() => Process
+    .start(p.join(sdkDir.path, 'bin', 'dartdevc'), ['--persistent_worker']));
```

Comment:
The `BazelWorkerDriver` manages the processes, it has some options for how many idle workers to keep alive and also max number of active workers to spawn, currently I am just using the defaults (4 for both).

All the processes are shut down when the pub process shuts down.